x-pack/solutions/security/plugins/security_solution/public/app/actions/toggle_column/cell_action/toggle_column.ts

```diff
@@ -69,11 +68,12 @@ export const createToggleColumnCellActionFactory = createCellActionFactory(
       }
 
       // When the flyout was initiated from an alerts table, use its toggleColumn action
-      if (metadata.alertsTableRef?.current && scopeId === TableId.alertsOnAlertsPage) {
+      // We can safely do this now as we do not have a page with multiple alerts tables. If that were the case we'd have problem with the ref
```

Comment:
Although this comment is useful for a PR review, I think it might be confusing for a code reading this file as you are just checking if the ref has a value here before using it 🤔 

I would probably omit the comment altogether as the code is now self-explanatory – what do you think? ☺️